ddp-workspace/projects/ddp-pancan/src/styles.scss

```diff
@@ -69,13 +72,16 @@ a {
 }
 
 .Link {
-    color: #7154ff;
+    color: $primary-color;
 }
 
 // TODO: borders below for debug reason. Remove when header and footer are ready
-header {
-    border-bottom: 1px solid black;
-}
 footer {
     border-top: 1px solid black;
 }
+
+.SimpleButton {
+    @extend .btn-common;
```

Comment:
#minor
suggest moving the styles to line 59 (as to avoid duplication)

Reply:
oops, didn't see it, fixed!